### PR TITLE
Update manual testing instructions in DG

### LINKED ISSUE #372: [PE-D][Tester E] Manual Testing Instructions: Potential wrong instructions for opening JAR file
Under "Instructions for manual testing", the instructions are that a user should launch the app by doubling clicking the JAR file. 

But I think this is not what users are supposed to do? The instructions on the CS2103T website state that users are supposed to open the file using `java -jar` command rather than double clicking. Double clicking also doesn't actually launch the app for me because I get an error "A Java Exception occurred". But this error does not happen when I run `java -jar` in the terminal. 

-------------
Labels: `type.DocumentationBug` `severity.Low`
original: tingxuanp/ped#14

### PR DESCRIPTION
**Pull request by seah-minlong**
Fixes #372

### CODE REVIEW

File: docs/DeveloperGuide.md [hunk 1]
```diff
@@ -459,19 +459,19 @@ testers are expected to do more *exploratory* testing.
 
 1. Initial launch
 
-   1. Download the jar file and copy into an empty folder
+   1. Download the jar file and copy into an **empty folder** in which the app is allowed to create files.
 
-   1. Double-click the jar file Expected: Shows the GUI with a set of sample contacts. The window size may not be optimum.
+   2. Open a command window. Run the `java -version` command to ensure you are using Java 17.
+
+   3. Launch the jar file using the `java -jar` command. The window size may not be optimum.
```
Comment (Justin-Yeo) [inline docs/DeveloperGuide.md:466]: should we specify like `java -jar contactsforgood.jar`

File: docs/DeveloperGuide.md [hunk 2]
```diff
@@ -459,19 +459,19 @@ testers are expected to do more *exploratory* testing.
...
 
 1. Saving window preferences
 
    1. Resize the window to an optimum size. Move the window to a different location. Close the window.
 
-   1. Re-launch the app by double-clicking the jar file.<br>
+   2. Re-launch the app using the `java -jar` command.<br>
```
Comment (Justin-Yeo) [inline docs/DeveloperGuide.md:472]: same as above comment

**Review by Justin-Yeo — CHANGES_REQUESTED** [review]

File: docs/DeveloperGuide.md (hunk 1 above)
Comment (seah-minlong) [reply docs/DeveloperGuide.md]: Sounds good will make the change

**Review by Justin-Yeo — APPROVED** [review]
LGTM

**Pull request merged**